src/View/Helper/PaginatorHelper.php

```diff
@@ -410,12 +417,22 @@ public function sort($key, $title = null, array $options = [])
         $locked = isset($options['lock']) ? $options['lock'] : false;
         unset($options['lock']);
 
-        $sortKey = $this->sortKey($options['model']);
         $defaultModel = $this->defaultModel();
+        $incomingKey = $key;
+        $model = Hash::get($options, 'model', $defaultModel);
+        list($table, $field) = explode('.', $key . '.');
```

Comment:
Should this limit to 2 pieces? I could forsee someone using `.` in their field like a joker.